Tweak.xm

```diff
@@ -254,7 +299,9 @@ CFStringRef (*_dynamic_WiFiNetworkGetProperty)(void *, CFStringRef) = (CFStringR
 
 - (void)applyAlternateActionForSwitchIdentifier:(NSString *)identifier
 {
-	if ([identifier isEqualToString:@"com.a3tweaks.switch.wifi"])
+    LoadSettings();
```

Comment:
I don't understand why you are loading the settings in these callbacks and the long-press callback too. Isn't this unnecessary?